scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -0,0 +1,152 @@
+# Pre-processing single-cell RNA-seq data 
+
+**CCDL 2019**
+
+#### In this section, we will be running through the basics of pre-processing
+single-cell RNA-seq data.
```

Comment:
This needs to go on the line above to render properly.

Reply:
Because this is a Markdown file, you can use the "one sentence per line" system rather than the 80 character per line limit.